redislite/client.py

```diff
@@ -113,28 +113,25 @@ def _start_redis(self):
         )
         fh.close()
 
-        command = 'redis-server %s' % config_file
-        logger.debug('Running: %s', command)
-        rc = os.system(command) >> 8
+        command = ['redis-server', config_file]
+        logger.debug('Running: %s', ' '.join(command))
+        rc = subprocess.call(command)
         if rc:  # pragma: no cover
             raise RedisLiteException('The binary redis-server failed to start')
 
         # Wait for Redis to start
         timeout = True
-        for i in range(0, self.start_timeout):
+        for i in range(0, self.start_timeout*10):
             if os.path.exists(self.socket_file):
                 timeout = False
                 break
-            time.sleep(1)
+            time.sleep(.1)
```

Comment:
maybe start_timeout and the sleep time should be configurable from configuration file?